tasks/src/reinforcement-learning/about.md

```diff
@@ -23,13 +27,15 @@ You can contribute variants of this task [here](https://github.com/huggingface/h
 **Environment:** The part of the world the agent interacts, comprising everything outside the agent.
 
 
-**State:** Information the agent receives from the environment. In the case of a video game it can be a frame (a screenshot), in the case of a chess playing agent it can be the board position, in the case of a trading agent it can be the price of a certain stock.
+**State:** It is the summary of information presented as the function of history of actions,observations and rewards . In the case of a video game it can be a frame (a screenshot), in the case of a chess playing agent it can be the board position, in the case of a trading agent it can be the price of a certain stock.

```

Comment:
I don't agree with this definition. State is rather the current situation, e.g. for chess it can be the position of pieces on the board.

Reply:
This definition is quite puzzling for me let's keep that simple but let's put the difference between state and obserrvation:
https://huggingface.co/blog/deep-rl-intro#observationsstates-space

Observations/States are the information our agent gets from the environment. In the case of a video game, it can be a frame (a screenshot). In the case of the trading agent, it can be the value of a certain stock, etc.

State s: is a complete description of the state of the world (there is no hidden information). In a fully observed environment.

Observation o: is a partial description of the state. In a partially observed environment.